screenpy/settings.py

```diff
@@ -10,5 +10,6 @@
 INDENT_CHAR = " "
 INDENT_SIZE = 4
 
+UNABRIDGED_LOGGING = False
```

Comment:
I think this should be `UNABRIDGED_NARRATION`, since that's kind of our standin for "logging". I'm not sure what adapters users will have connected; "logging" might not accurately describe them all.